lists.js

```diff
@@ -398,6 +398,22 @@ List.prototype.becomeLinked = function () {
     }
 };
 
+List.prototype.asJSON = function () {
```

Comment:
Because a List can't be converted to JSON, unlike an Array. Thus, if you have a List of Lists, and call stringify on the outer one, the inner one will be serialized as "A List [1,2,3,4]".